docker-compose.yml

```diff
@@ -20,11 +20,24 @@ services:
     <<: *netbox
     depends_on:
     - redis
+    - postgres
     entrypoint:
+    - "/sbin/tini"
+    - "--"
+    command:
     - /opt/netbox/venv/bin/python
     - /opt/netbox/netbox/manage.py
-    command:
     - rqworker
+  netbox-housekeeping:
+    <<: *netbox
+    depends_on:
+    - redis
+    - postgres
+    entrypoint:
+    - "/sbin/tini"
```

Comment:
Same here; IMO the entrypoint can just be `/bin/sh` or the script itself.

Reply:
Here is good [explanation](https://blog.phusion.nl/2015/01/20/docker-and-the-pid-1-zombie-reaping-problem/) why an "init" process should be PID 1.

The housekeeping script is starting a new process for every check. So I think it is good to have a PID 1 that can manage hanging or zombie processes correctly.

Reply:
The housekeeping script is waiting for the command to return, so no risk of zombies there.

Reply:
If you think `tini` is a good addition, wouldn't it then make sense to make it our default entrypoint? Like you proposed earlier, right? If we do it carefully, I don't think the chance of breaking any existing configs is too high, i.e. that we just change the `entrypoints` and not the `cmd` sections. Wdyt?

Reply:
Yes, I think it would be a good addition. 
To preserve the current behaviour I would change the `ENTRYPOINT ` to `[ "/sbin/tini", "--" ]` and the `CMD` to `[ "/opt/netbox/docker-entrypoint.sh", "/opt/netbox/launch-netbox.sh" ]`. Then we can replace the CMD in the `docker-compose.yml` for the worker and housekeeping containers. With this the entrypoint script would only run on the "main" container, like it does now. It would also be easy for users to add a new worker (plugins can need additional workers) by replacing only the command in the `docker-compose.override.yml` file.